caluma/caluma_form/models.py

```diff
@@ -245,12 +285,20 @@ class Meta:
 class File(UUIDModel):
     name = models.CharField(max_length=255)
 
+    def _get_file_names(self):
+        old_file = self.history.first()
+        new_name = f"{old_file.pk}_{old_file.name}"
+        return (self.object_name, new_name)
+
     def _move_blob(self):
         # move the file on update
         # this makes sure it stays available when querying the history
-        old_file = self.history.first()
-        new_name = f"{old_file.pk}_{old_file.name}"
-        client.move_object(self.object_name, new_name)
+        client.move_object(*self._get_file_names())
+
+    def copy(self, *args, **kwargs):
```

Comment:
This does copy the file in minio, not the `File` object. So for consistency, it should be called `copy_blob()`.